book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative('./rental')
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] The getter and setter (`attr_accessor`) for `:rental` should be present, not `attr_reader`. Kindly fix this.